consensus/types/proto/state_sync.proto

```diff
@@ -30,6 +30,25 @@ message GetBlockResponse {
     consensus.Block block = 2; // The block being provided to the peer
 }
 
+enum StateSyncMessageType {
```

Comment:
https://github.com/pokt-network/pocket/issues/258

Reply:
thanks for sharing, updated.